doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,451 @@
+header itself, we need in total 8 additional bytes compared to a design that uses a path-type.
+
+Separate Identifier option
+---------------------------
+
+Instead of just having a single FABRID HBH option, we decided to move the packet ID and packet timestamp to another HBH option,
+the so called Identifier option, because this might also be useful for other HBH extensions and not just for FABRID
+(e.g., it would allow to port EPIC-HP from a path type to a HBH extension).
+Since FABRID still requires the packetID and packet timestamp, providing the Identifier option became mandatory for FABRID packets.
+The cost of moving the Identifier to a separate HBH extension is 4 bytes, where 2 bytes are used for the HBH option type and length
+fields and 2 bytes for padding to have the FABRID HBH extension 4 bytes alligned.
+
+Length of PacketID and PacketTimestamp for the Identifier HBH option
+---------------------------------------------------------------------
+
+The Identifier has a timestamp with a length of 27 bits, which encodes the relative time in milliseconds after
+the timestamp value of the first InfoField of the SCION header.
+The 27 bit allow to save relative timestamps with a difference of up to 37 hours which fulfills the requirement
+that a path can be valid for up to 24 hours.
+
+Length of FABRID policyID and how to determinte whether policy is local or global
+----------------------------------------------------------------------------------
+
+The decision on whether a certain FABRID policy is a local or global policy is done by the control service,
+hence we do not have to reserve any bits of the FABRID policy index in the FABRID packets to encode whether
+it is a local or global policy.
+In the header design the FABRID policyIndex has a length of 1 byte, which allows 256 different options.
+But since the control service can configure the policies per interface pair and / or per IP range, there
+are many more options than the 256.
+
+
+Compatibility
+===============
+
+FABRID is a new extension which uses the SCION Hop-by-Hop extension which allows
+for incremental deployment of FABRID.
+If a border router does not understand the FABRID Hop-by-Hop extension
+it will simply ignore it and hence not provide any of the FABRID functionality and forward the packet as if it
+is a normal SCION packet.
+The "FABRID enabled" flag allows the sending endhost to choose for which ASes to enable path validation,
+but is also specifically required, for edge-cases where an AS just starts to roll out FABRID functionality to its
+border routers, but the sender does not yet know that this AS is FABRID aware.
+Because if a packet does not contain a HVF for this AS, the packet has to be dropped unless the sender can explicitly
+state that FABRID is disabled for this hop.
+
+Implementation
+================
+
+We plan to provide the implementation in the following steps (PRs):
```

Comment:
Can we separate into what has been done already vs what will be done on a second stage?

Reply:
I tried to seperate it into what code is (almost) available for a merge and what will be done in a second stage.